src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  --header-link-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+img {
+  display: block;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
```

Comment:
Don't use tag names for styling (except html and body)
<img width="819" alt="image" src="https://github.com/user-attachments/assets/6c97d1cc-07d1-44a0-a48a-472b78aabf92">